email_mailbox/src/components/SettingAccountWrapper.js

```diff
@@ -828,6 +856,18 @@ class SettingAccountWrapper extends Component {
     }
   };
 
+  handleConfirmDeleteCustomDomain = async () => {
+    if (!this.state.deleteCustomDomainsParams) return;
+    const { domain } = this.state.deleteCustomDomainsParams;
+    const res = await deleteDomain(domain); //api
+    if (!res || res.status !== 200) return;
```

Comment:
no error handling/feedback if user can't delete domain?

Reply:
Solved in new commit